keyboards/crkbd/keymaps/mcrown/config.h

```diff
@@ -0,0 +1,98 @@
+/** @file config.h
+ *  @brief config header that includes function prototypes and external variables.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * @author Mario Corona (mariocc@comunidad.unam.mx) 2021
+ *
+ */
+
+#pragma once
+
+#define FORCE_NKRO
+
+/* Select hand configuration */
+#define MASTER_LEFT
+
+/* Comment this definition out if you want the main display to be output in horizontal mode */
+#define OLED_VERTICAL
+
+#define USE_SERIAL_PD2
+
+#define TAPPING_TERM 200
+
+#define MASTER_LEFT
+
+#define USE_SERIAL_PD2
+
```

Comment:
Some of these are duplicate, or deprecated. 
```suggestion
#define TAPPING_TERM 200
```